README.md

```diff
@@ -30,5 +30,322 @@
 
 # Triton Inference Server
 
-**NOTE: You are currently on the r22.03 branch which tracks stabilization
-towards the next release. This branch is not usable during stabilization.**
+Triton Inference Server provides a cloud and edge inferencing solution
+optimized for both CPUs and GPUs. Triton supports an HTTP/REST and
+GRPC protocol that allows remote clients to request inferencing for
+any model being managed by the server. For edge deployments, Triton is
+available as a shared library with a C API that allows the full
+functionality of Triton to be included directly in an
+application.
+
+## What's New in 2.20.0
+- Models can now [load from a serialized model_config message](docs/protocol/extension_model_repository.md#load) with the Triton Server API.
```

Comment:
In the readme.md, lines should not be more than 80 characters. You will need to break them up. Also, you will need to add empty lines after headers and between each '-'.

Reply:
Done